src/Range.jsx

```diff
@@ -261,23 +261,24 @@ class Range extends React.Component {
     return true;
   }
 
-  trimAlignValue(v, nextProps = {}) {
+  trimAlignValue(v, handle = this.state.handle, nextProps = {}) {
```

Comment:
`createSlider` will call `trimAlignValue` too https://github.com/react-component/slider/search?utf8=%E2%9C%93&q=trimAlignValue&type=

Reply:
Yes, but args `handle` & `nextProps` both optional. So,  I think it dosen't matter.